### DISCUSSION TITLE
Home Assistant OS VM

### DISCUSSION THREAD

**Discussion by jerome6994** [tteck/Proxmox#1001] [Ideas]
Hello
Great work that these different script
Is it possible on the script for setting up a VM for Home Assistant to have the choice of disk size in order to choose, for example, 64GB instead of 32GB today?
Thank you very much hoping this is possible as it simplifies disk expansions which I never manage to produce
Jérôme

**Comment by tteck** [comment]
Increasing isn't a problem.

![Screenshot 2023-01-31 3 05 54 AM](https://user-images.githubusercontent.com/21342670/215703151-4505cbc0-85f0-43de-976d-ff51290fc12a.png)

That will add 32GB

**Reply by tteck** [reply]
But, to answer your question, no, the disk image sets the initial disk size.

**Comment by jerome6994** [comment]
Going through proxmox to extend disk space is a step but then you have to extend the partition and that's often where I miss everything and don't have to understand everything
But thanks for the reply
